### ISSUE TITLE
A few questions

### ISSUE THREAD

**Issue by SUBmarino006** [Staphylococcus/federal-eu-constitution#36]
Hello, I am SUBmarino006 and I am new here. I would love to help in the creation of this project in whatever I can. I have started reading the first file and I have a few questions and suggestions that may be useful for a better clarity. 

-The Article II includes a general statement regarding freedoms, including (but not limited to) the freedom of press, religion or assembly. These are expanded through in the Articles III, IV and V. Wouldn't it make sense to transform them in Articles II.I, II.II and II.III? I don't think it is as necessary to have a many-articled constitution as much as a practical one in its legal form. 

-Article IX says that "In addition, the only searches that shall be considered lawful [...] supported by the consent of the legal owner, over eighteen years of age [...]" Why is the age relevant for this? If a teenager is considered to be legally responsible, he might as well give consent (or their parents should).

-Articles XVI and XVII state that: 
Article XVI.
No one may be extradited except by virtue of a duly ratified international convention.
Article XVII.
Under no circumstances may a political refugee legally entitled to asylum be handed over or extradited.

I suppose that Article XVII is predominant over Article XVI in case of confrontation. Therefore, shouldn't they be inversed, for they are of a bigger importance?

Thank you for attending my questions. I am looking forward being answered.

**Comment by Mathew-Iron** [comment]
Hello @SUBmarino006 

"-The Article II includes a general statement regarding freedoms, including (but not limited to) the freedom of press, religion or assembly. These are expanded through in the Articles III, IV and V. Wouldn't it make sense to transform them in Articles II.I, II.II and II.III? I don't think it is as necessary to have a many-articled constitution as much as a practical one in its legal form."

It's true that grouping them together would reflect their common conceptual basis and streamline the document, but on the other hand, keeping them separate could ensure that each right is clearly articulated and can be individually referenced without confusion, and emphasize the protection of each right.

Personally, I think it's better to have more articles dealing specifically with a freedom.

I made a pull request , in this one, I've tried to make each article less general and more specific and precise. For example, Article II has been reworked to focus on freedom of expression, conscience and thought, because the other articles that follow already deal with other freedoms.

"-Article IX says that "In addition, the only searches that shall be considered lawful [...] supported by the consent of the legal owner, over eighteen years of age [...]" Why is the age relevant for this? If a teenager is considered to be legally responsible, he might as well give consent (or their parents should)."

This is a very fair comment, which could help us to give our Member States a certain amount of latitude and allow them to define the legal age they want.

We should reword it as follows: “In addition, the only searches that shall be considered lawful [...] supported by the consent of the legal owner, considered by law to be legally responsible [...]”.

In this regard, I also think we should add that a person considered by law to be legally responsible for his or her actions being legally on the property with the consent of the property owner can consent to a search if the legal owner is not available.

In line with this, it would also be advisable to add an article specifying who can be considered legally responsible (without specification, a member state could define the age of 5 as legally responsible, there is a possible debate).

"-Articles XVI and XVII state that:
Article XVI.
No one may be extradited except by virtue of a duly ratified international convention.
Article XVII.
Under no circumstances may a political refugee legally entitled to asylum be handed over or extradited."

That's right, we should reword it to clarify what happens in the event of a confrontation.


Thanks for your help, you have a lot of interesting ideas.

**Comment by SUBmarino006** [comment]
Another issue with Article XVI-XVII: You shouldn't be able to extradite somebody who may be condemned to death for their crime, independently of whether they are a political refugee or not. At least in my home country of Spain, this is illegal. 

Also, I would like to talk about the elephant in the room: we should really specify that the constitution is binding to every single person laying physically or juridically in European territory, without exceptions due to ideology, religion or culture: Everybody must actively accept that every human is equal, that slavery is illegal, etc... If somebody isn't ready to accept this fact, they might as well not be allowed to live here, no matter if they are a citizen, or even if they have written this text. This article is, I believe, essential for the survival of the Union. We can't be tolerant to intolerance.

**Comment by Mathew-Iron** [comment]
@SUBmarino006 "You shouldn't be able to extradite somebody who may be ...." Yes, I think we should add that, I'm not sure but I think it's also the case in France, in any case we should add that.

"Also, I would like to talk abo..." I mean, the constitution is the supreme law and breaking the law leads the government to punish you, so we're planning to add an article that says the constitution is the supreme law of the land (but it won't be in the freedom part)

"human is equal, that slavery is illegal, " : Article VII/Article VIII.
